model/task/task.go

```diff
@@ -68,12 +68,13 @@ type Task struct {
 	FinishTime    time.Time `bson:"finish_time" json:"finish_time"`
 	ActivatedTime time.Time `bson:"activated_time" json:"activated_time"`
 
-	Version           string `bson:"version" json:"version,omitempty"`
-	Project           string `bson:"branch" json:"branch,omitempty"`
-	Revision          string `bson:"gitspec" json:"gitspec"`
-	Priority          int64  `bson:"priority" json:"priority"`
-	TaskGroup         string `bson:"task_group" json:"task_group"`
-	TaskGroupMaxHosts int    `bson:"task_group_max_hosts,omitempty" json:"task_group_max_hosts,omitempty"`
+	Version           string             `bson:"version" json:"version,omitempty"`
+	Project           string             `bson:"branch" json:"branch,omitempty"`
+	Revision          string             `bson:"gitspec" json:"gitspec"`
+	Priority          int64              `bson:"priority" json:"priority"`
+	TaskGroup         string             `bson:"task_group" json:"task_group"`
+	TaskGroupMaxHosts int                `bson:"task_group_max_hosts,omitempty" json:"task_group_max_hosts,omitempty"`
+	Logs              apimodels.TaskLogs `bson:"logs,omitempty" json:"logs,omitempty"`
```

Comment:
is this the first case of us having a dependency on apimodels in the models package? I think it's fine but I wanted to make sure.

Reply:
nope, the package was already imported (it's part of the task end detail)